policyengine_us/variables/gov/states/la/tax/income/exempt_income/la_agi_exempt_income.py

```diff
@@ -6,40 +6,39 @@ class la_agi_exempt_income(Variable):
     entity = TaxUnit
     label = "Louisiana income that is exempt from the adjusted gross income"
     defined_for = StateCode.LA
-    unit = USD
+    reference = "https://revenue.louisiana.gov/TaxForms/IT540i(2021)%20Instructions.pdf#page=9"
     definition_period = YEAR
 
     # Functions as subtractions.
     def formula(tax_unit, period, parameters):
         p = parameters(period).gov.states.la.tax.income.exempt_income
         total_exempt_income = add(tax_unit, period, p.sources)
         if p.reduction.in_effect:
-            # Option 1
-            exempt_income_reduction = p.reduction.rate.calc(
-                total_exempt_income
-            )
-            # Option 2
+            # Option 1 is to reduce the exempt income by a marginal rate
+            reduced_exempt_income = p.reduction.rate.calc(total_exempt_income)
+            # Option 2 is to reduce the federal tax deduction by the percentage of
+            # exempt income to AGI
             agi = tax_unit("adjusted_gross_income", period)
             exempt_income_rate = np.zeros_like(agi)
             mask = agi != 0
             exempt_income_rate[mask] = total_exempt_income[mask] / agi[mask]
-            itemizes = tax_unit("tax_unit_itemizes", period)
-            itemized_deductions = tax_unit("la_itemized_deductions", period)
-            claimed_itemized_deductions = itemizes * itemized_deductions
-            fed_tax_deduction = tax_unit("la_federal_tax_deduction", period)
-            pre_exempt_income_tax = max_(
-                agi - claimed_itemized_deductions - fed_tax_deduction, 0
+            # The second option only applies if the tax unit has a federal tax deduction
```

Comment:
Could you spell this out more? I see it references instructions for Line 9, is that where this is coming from? Line and page numbers may help.

Reply:
Lmk what you think